docs/how_to/use_sampler.rst

```diff
@@ -0,0 +1,239 @@
+#####################################################
+Compute circuit output probabilities with ``Sampler``
+#####################################################
+
+This guide shows how to get the probability distribution of a quantum circuit with the :class:`~qiskit.primitives.Sampler` primitive.
+
+.. note::
+
+    While this guide only uses Qiskit Terra's implementation of the ``Sampler`` primitive, there are other
+    implementations of this primitive like Qiskit Terra's :class:`~qiskit.primitives.BackendSampler`, Qiskit Aer's :class:`~qiskit_aer.primitives.Sampler`
+    and Qiskit Runtime's :class:`~qiskit_ibm_runtime.Sampler`.
+
+Initialize quantum circuits
+===========================
+
+The first step is to create the :class:`~qiskit.circuit.QuantumCircuit` from which you want to obtain the probability distribution.
+
+.. plot::
+    :include-source:
+
+    from qiskit import QuantumCircuit
+
+    qc = QuantumCircuit(2)
+    qc.h(0)
+    qc.cx(0,1)
+    qc.measure_all()
+    qc.draw("mpl")
+
+.. testsetup::
+
+    from qiskit import QuantumCircuit
+
+    qc = QuantumCircuit(2)
+    qc.h(0)
+    qc.cx(0,1)
+    qc.measure_all()
+
+.. note::
+
+    The :class:`~qiskit.circuit.QuantumCircuit` you pass to :class:`~qiskit.primitives.Sampler` has to include measurements.
+
+Initialize the ``Sampler``
+==========================
+
+Then, you need to create a :class:`~qiskit.primitives.Sampler` object.
+
+.. testcode::
+
+    from qiskit.primitives import Sampler
+
+    sampler = Sampler()
+
+Run and get results
+===================
+
+Now that you have defined ``sampler``, you can create a :class:`~.PrimitiveJob` (subclass of :class:`~qiskit.providers.JobV1`) with the
+:meth:`~qiskit.primitives.Sampler.run` method and, then, you can get the results (as a :class:`~qiskit.primitives.SamplerResult` object) with
+the results with the :meth:`~qiskit.providers.JobV1.result` method.
```

Comment:
Same as with the estimator, I don't think that "you can create a job" is obscuring what you are actually doing (which is running the sampler)

```suggestion
Now that you have defined your ``sampler``, you can run it by calling the :meth:`~qiskit.primitives.Sampler.run` method, 
which returns an instance of :class:`~.PrimitiveJob` (subclass of :class:`~qiskit.providers.JobV1`). You can get the results from the job (as a :class:`~qiskit.primitives.SamplerResult` object) 
with the :meth:`~qiskit.providers.JobV1.result` method.
```